src/lib/index.ts

```diff
@@ -25,33 +25,28 @@ type HttpQueue = {
     drain: (timeout?: number) => Promise<void>
 };
 
-const createHttpQueue = (): HttpQueue => {
+type HttpQueueOptions = {
+    forceXHR?: boolean
+};
+
+const createHttpQueue = (options: HttpQueueOptions): HttpQueue => {
     const store = createStore({ reducer, middlewares });
 
     /**
      * Schedules a HTTP request.
-     *
-     * @param {IRequest} request The request definition
-     * @returns void
-     *
      */
     const schedule = (manifest: IRequest) => {
         store.dispatch(
             add({
+                forceXHR: !!options.forceXHR,
                 id: numericRandomId(),
                 ...manifest,
             })
         );
     };
 
     /**
-     * Drains the queue and resolves when the queue is empty OR a
-     * given timeout is exceeded.
-     *
-     * @param {number} timeout Resolves the promise after timeout has exceeded
-     *
-     * @returns {Promise<void>}
-     *
+     * Drains the queue and resolves when the queue is empty OR a given timeout is exceeded.
      */
     const drain = async (timeout: number = 3000) => {
```

Comment:
return type

Reply:
why? it's an async function without a return. isn't Promise<void> then the implicit return type?

Reply:
I think @ansgarS suggested an explicit return type

Reply:
and I talked to him and he agreed that it's not necessary as it's implicit